test/feeds.test.js

```diff
@@ -62,3 +62,65 @@ describe('test /feeds/:id responses', () => {
     expect(res.body instanceof Array).toBe(false);
   });
 });
+
+describe('test POST /feeds endpoint', () => {
+  const createdItems = 5;
+  const feedData = {
+    author: 'foo',
+    url: 'http://telescope200.cdot.systems',
+  };
+
+  // Array of feeds
+  const feeds = [...Array(createdItems).keys()].map(item => {
+    return new Feed('foo', `http://telescope${item}.cdot.systems`);
+  });
+
+  beforeAll(() => Promise.all(feeds.map(feed => feed.save())));
```

Comment:
If you're going to manually add feeds in every test (using POST) you might not need this

Reply:
True. But do tests run in parallel or in order? If in order I don't need this. In parallel I would.

Reply:
This part only adds feeds using `feed.save()`, which is not very useful to you since you want to do `POST` and test `json`, `url`, `author`. I think maybe creating 3 Feed objects (one is ok, one without author, one without url) and using them in the specific test could be all you need.